config-provisioning/src/main/java/com/yahoo/config/provision/zone/ZoneFilter.java

```diff
@@ -16,24 +15,13 @@ public interface ZoneFilter {
     /** Negates the next filter. */
     ZoneFilter not();
 
-    /** Zones which are upgraded by the controller. */
```

Comment:
Remember to update implementation in hosted

Reply:
Yes :)